pkgs/development/libraries/glibc/common.nix

```diff
@@ -50,7 +50,7 @@ assert withLinuxHeaders -> linuxHeaders != null;
 assert withGd -> gd != null && libpng != null;
 
 stdenv.mkDerivation ({
-  inherit version;
+  version = version + patchSuffix;
```

Comment:
Potentially this breaks downstream usage of the version attribute — it should be fine though if `lib.version*` was used.